router/core/executor.go

```diff
@@ -189,19 +189,37 @@ func (b *ExecutorConfigurationBuilder) Build(ctx context.Context, opts *Executor
 }
 
 func buildNatsOptions(eventSource config.NatsEventSource, logger *zap.Logger) ([]nats.Option, error) {
-
 	opts := []nats.Option{
```

Comment:
I'd add `nats. AllowReconnect(true)`

Reply:
To ensure the default? Is is enabled by default